frontend/src/components/SymptomForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router";
+import { useDispatch, useSelector } from "react-redux";
+import { me } from "../store/authSlice";
+import { addSymptomEntry } from "../store/symptomSlice";
+import Sidebar from "./Sidebar";
+
+const SymptomForm = () => {
+  const [date, setDate] = useState("");
+  const [time, setTime] = useState("");
+  const [symptom, setSymptom] = useState("");
+  const [severity, setSeverity] = useState("");
+
+  const user = useSelector((state) => state.auth.me);
+  console.log(user.username);
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const handleSymptomSubmit = async (event) => {
+    event.preventDefault();
+    await dispatch(
+      addSymptomEntry({
+        username: user.username,
+        date: date,
+        time: time,
+        symptom: symptom,
+        severity: severity,
+      })
+    );
+    navigate("/profile");
+  };
+
+  useEffect(() => {
+    dispatch(me());
+  }, [dispatch]);
+
+  const symptoms = [
```

Comment:
Similar to a comment about the food options - this should be pulled from the database